vagrant.yml

```diff
@@ -0,0 +1,4 @@
+- hosts: all
+  roles:
+    - role: vagrant
+      iface: eth1
```

Comment:
It seems to me that setting up hosts files would be generally useful; even outside of using Vagrant.  This role should be split off into a new PR, maybe `hosts-file`?